templates/pg_dump.sh.epp

```diff
@@ -1,3 +1,20 @@
+<%- |
+  Boolean                                  $compress,
+  Array                                    $databases,
```

Comment:
You could use `Array[String[1]] $databases = []` and simplify the code. Right now it still deals with `undef` and it not being an `Array`.